Mathlib/Data/Fintype/EquivFin.lean

```diff
@@ -395,6 +395,13 @@ theorem card_lt_of_surjective_not_injective [Fintype α] [Fintype β] (f : α 
 
 end Fintype
 
+/-- Choose an arbitrary linear order on a `Fintype`: this is not an instance because in most
+situations, choosing a linear order extending a given preorder, or a particular linear order
+is preferred over choosing *any* linear order. -/
+protected noncomputable def Finite.linearOrder {α : Type*} [Finite α] : LinearOrder α :=
+  haveI := Fintype.ofFinite α
+  LinearOrder.lift' _ (Fintype.equivFin α).injective
```

Comment:
I guess arguably we should rename `Fintype.equivFin` to `Finite.equivFin`, since the former is very non-constructive anyway.

Reply:
Thanks for pointing this out; I hadn't realised. I agree that this PR should be closed.